algorithms/sorting/bubble-sort/BubbleSort.java

```diff
@@ -1,10 +1,26 @@
 /**
- * A bubble sort implementation in java
+ * A bubble sort implementation for int arrays in java
  * 
  * @author Harry Willis
  */
  public class BubbleSort {
-    public void bubbleSort(int[] a) {
-        
+    public static void bubbleSort(int[] a) {
```

Comment:
Comment your function!

What are the arguments? What does the function do?

Also, single letter names are pretty hard to parse. Maybe the param should be called `array` for clarity.